sql/README.md

```diff
@@ -21,3 +21,4 @@ Once you are ready to add a new query to the repository, open a pull request fol
 ### 2022/11
 
 * [Performance Lab plugin version distribution in a given month](./2022/11/performance-lab-version-distribution.sql)
+* [Core Web Vital "good" rates by WordPress version.](./12/cwvs-by-wordpress-version.sql)
```

Comment:
Since it's December now, can you put this in a new section `2022/12` above this one instead?

Also please review the link regarding the file location (see above).

Reply:
updating to 2023/01